source/voice_control/assign_areas_floors.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: "Assigning devices to areas and areas to floors"
+related:
+  - docs: /docs/organizing/areas/
+    title: Areas
+  - docs: /docs/organizing/floors/
+    title: Floors
+  - docs: /voice_control/custom_sentences/
+    title: Assist - custom sentences
+  - docs: /voice_control/best_practices/
+    title: Best practices with Assist
+---
+
+Another best practice with Assist is to create an architecture of areas and floors in your home, since it will make it consistent and easy to understand for Assist.
+
+### To create missing areas 
+
+1. Go to Settings > Areas, Labels & Zones
+2. Click on Create area
```

Comment:
```suggestion
2. Select **Create area**.
```
- We use [select instead of click](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)
- We use bold to markup UI elements